modules/repository-gcs/src/main/java/org/elasticsearch/repositories/gcs/GoogleCloudStorageBlobStore.java

```diff
@@ -380,7 +386,7 @@ public void write(byte[] b, int off, int len) throws IOException {
                                 while (written < len) {
                                     // at most write the default chunk size in one go to prevent allocating huge buffers in the SDK
                                     // see com.google.cloud.BaseWriteChannel#DEFAULT_CHUNK_SIZE
```

Comment:
Nit: we can probably remove the `see ...` comment because it's proper javadoc above now